provelo_analytic_account/models/sale_order.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.multi
+    def _prepare_invoice(self):
+        values = super(SaleOrder, self)._prepare_invoice()
+        journal = self.location_id.journal_id
```

Comment:
I see that in our resource_activity module, a sale order is extended to have an activity (not required) and a location from that activity (related field). That mean's we're not sure there's a `location_id` if I understand it correctly, so we should test for self.location_id before accessing it's values.

Reply:
👍